src/main.cc

```diff
@@ -1782,6 +1796,16 @@ masterShutdownStart()
     shutting_down = 1;
 }
 
+/// Initiates restarting and reviving all kids.
```

Comment:
The code does not seem to initiate anything. Perhaps this method is missing a clarifying comment, somewhat similar to the one in masterReviveKids()?

Reply:
Rewrote.